src/Fabulous.XamarinForms/Views/Controls/Button.fs

```diff
@@ -47,9 +104,120 @@ type ButtonModifiers =
+                      | None -> ValueNone
+                      | Some v -> ValueSome v }
+            )
+        )
+
+    [<Extension>]
+    static member inline borderWidth(this: WidgetBuilder<'msg, #IButton>, value: float) =
+        this.AddScalar(Button.BorderWidth.WithValue(value))
+
+    [<Extension>]
+    static member inline padding(this: WidgetBuilder<'msg, #IButton>, value: Thickness) =
+        this.AddScalar(Button.Padding.WithValue(value))
+
+    [<Extension>]
+    static member inline padding(this: WidgetBuilder<'msg, #IButton>, value: float) =
+        ButtonModifiers.padding (this, Thickness(value))
+
+    [<Extension>]
+    static member inline padding
+        (
+            this: WidgetBuilder<'msg, #IButton>,
+            left: float,
+            top: float,
+            right: float,
+            bottom: float
+        ) =
+        ButtonModifiers.padding (this, Thickness(left, top, right, bottom))
+
+    [<Extension>]
+    static member inline characterSpacing(this: WidgetBuilder<'msg, #IButton>, value: float) =
+        this.AddScalar(Button.CharacterSpacing.WithValue(value))
+
+    [<Extension>]
+    static member inline contentLayout(this: WidgetBuilder<'msg, #IButton>, value: ContentLayout) =
+        let imagePos =
+            match value.Position with
+            | ImagePosition.Left -> Xamarin.Forms.Button.ButtonContentLayout.ImagePosition.Left
+            | ImagePosition.Right -> Xamarin.Forms.Button.ButtonContentLayout.ImagePosition.Right
+            | ImagePosition.Top -> Xamarin.Forms.Button.ButtonContentLayout.ImagePosition.Top
+            | ImagePosition.Bottom -> Xamarin.Forms.Button.ButtonContentLayout.ImagePosition.Bottom
+
+        this.AddScalar(Button.ContentLayout.WithValue(Button.ButtonContentLayout(imagePos, value.Spacing)))
+
+    [<Extension>]
+    static member inline font(this: WidgetBuilder<'msg, #IButton>, value: float) =
         this.AddScalar(Button.FontSize.WithValue(value))
 
     [<Extension>]
     static member inline font(this: WidgetBuilder<'msg, #IButton>, value: NamedSize) =
         this.AddScalar(Button.FontSize.WithValue(Device.GetNamedSize(value, typeof<Button>)))
+
+    [<Extension>]
+    static member inline font
+        (
+            this: WidgetBuilder<'msg, #IButton>,
+            ?size: float,
+            ?namedSize: NamedSize,
+            ?attributes: FontAttributes
+        ) =
+
+        let mutable res = this
+
+        match size with
+        | None -> ()
+        | Some v -> res <- res.AddScalar(Button.FontSize.WithValue(v))
+
+        match namedSize with
+        | None -> ()
+        | Some v -> res <- res.AddScalar(Button.FontSize.WithValue(Device.GetNamedSize(v, typeof<Button>)))
+
+        match attributes with
+        | None -> ()
+        | Some v -> res <- res.AddScalar(Button.FontAttributes.WithValue(v))
+
+        res
+
+    [<Extension>]
+    static member inline fontFamily(this: WidgetBuilder<'msg, #IButton>, value: string) =
```

Comment:
I'm thinking everything font related should be handle by one single `font` modifier (the one just above that function)

Reply:
Yeah as there are all optional if would work and less extensions to maintain 😀